anchor/common/ssv_types/src/operator.rs

```diff
@@ -0,0 +1,60 @@
+use crate::util::parse_rsa;
+use derive_more::{Deref, From};
+use openssl::pkey::Public;
+use openssl::rsa::Rsa;
+use std::cmp::Eq;
+use std::fmt::Debug;
+use std::hash::Hash;
+use types::Address;
+
+/// Unique identifier for an Operator.
+#[derive(Clone, Copy, Debug, Default, Eq, PartialEq, Hash, From, Deref)]
+pub struct OperatorId(pub u64);
+
+/// Client responsible for maintaining the overall health of the network.
+#[derive(Debug, Clone)]
+pub struct Operator {
```

Comment:
is there any benefit to tracking our cluster memberships here or is that being handled in the DB?

Reply:
IMO that should be handled in the db

Reply:
Yep, I recommend keeping Operator self contained and handling this in the DB